core-it-suite/src/test/resources/mng-7468-unsupported-params/config-build-execution/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.maven.its.mng7464</groupId>
+  <artifactId>maven-it-mng7468</artifactId>
+  <version>1.0</version>
+
+  <name>Maven Integration Test :: mng7468</name>
+  <description>
+    Test that ensures that warning about unsupported mojo parameters are generated.
+  </description>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.its.plugins</groupId>
+        <artifactId>maven-it-plugin-configuration</artifactId>
+        <version>2.1-SNAPSHOT</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>touch</goal>
+            </goals>
+            <configuration>
+              <invalidValue>value</invalidValue>
+              <invalidXml>
+                <item>value</item>
+              </invalidXml>
+            </configuration>
```

Comment:
This confuses me a bit. We don't validate the value, but only the param, no? So why not `invalidParam`? What am I missing?
The next block is an Xpp3Dom, so I guess `invalidXml` is correct.

Reply:
ok, `invalidParam` will be more precise - yes we check parameter names not values

`invalidXml` can be configuration for complex custom object or collections like list

I don't now  - can plugin have access to raw `Xpp3Dom` if not defined parameter in plugin.xml descriptor

Reply:
both new?
 - `invalidValue` -> `invalidParam`
 - ?

Reply:
done